packages/backend/src/app/helper/system/system.ts

```diff
@@ -104,28 +103,8 @@ const getEnvVar = (prop: SystemProp): string | undefined => {
 }
 
 export const validateEnvPropsOnStartup = async (): Promise<void> => {
-    const executionMode = system.get(SystemProp.EXECUTION_MODE)
-    const signedUpEnabled =
-        system.getBoolean(SystemProp.SIGN_UP_ENABLED) ?? false
     const queueMode = system.getOrThrow<QueueMode>(SystemProp.QUEUE_MODE)
-    const environment = system.get(SystemProp.ENVIRONMENT)
     await loadEncryptionKey(queueMode)
-
-    if (
```

Comment:
why not check for code execution type + sign up enabled?

Reply:
I think it's best to leave it up to the user's discern. This will be configurable using languages anyway so it doesn't make sense to disable signups if javascript is disabled, right?